packages/cli/src/commands/identity/test-domain-claim.ts

```diff
@@ -0,0 +1,57 @@
+import { flags } from '@oclif/command'
+import chalk from 'chalk'
+import { resolveTxt } from 'dns'
+import { ClaimCommand } from '../../utils/identity'
+
+export default class TestDomainClaim extends ClaimCommand {
+  static description =
+    'Tests whether the account has setup the TXT record required for a domain claim'
+  static flags = {
+    ...ClaimCommand.flags,
+    domain: flags.string({
+      required: true,
+      description: 'The domain you want to claim',
+    }),
+  }
+  static args = ClaimCommand.args
+  static examples = [
+    'test-domain-claim ~/metadata.json --domain test.com --from 0x47e172F6CfB6c7D01C1574fa3E2Be7CC73269D95',
+  ]
+  self = TestDomainClaim
+  async run() {
+    const res = this.parse(TestDomainClaim)
+    const metadata = this.readMetadata()
```

Comment:
How does this work with `metadata.json` of an account you don't control but want to verify?

Reply:
metadata is off-chain and the urls publicly available in blockscout, so you can get that and validate